src/Illuminate/Database/Query/Grammars/MySqlGrammar.php

```diff
@@ -113,6 +113,17 @@ protected function compileJsonContainsKey($column)
         return 'ifnull(json_contains_path('.$field.', \'one\''.$path.'), 0)';
     }
 
+    /**
+     * Compile a "JSON cast" statement into SQL.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    public function compileJsonCast($value)
+    {
+        return "cast($value as json)";
```

Comment:
for code consistency, I would switch to single quotes and concatenation, rather than double quotes and interpolation.

Reply:
Gotcha, will do!